### ISSUE TITLE
Unable to Install Node in to a z/OS 2.3 Environment

### ISSUE THREAD

**Issue by FrancisDBurgess** [ibmruntimes/node#114]
The z/OS Node JS installation logic has a hard prereq in it's SMPMCS (part of the installation logic) of FMID HLE77A0 which it turns is Language Environment (LE) and there are no supplied
jobs to allow you to create your own SMP/E environment, so the simplest and most unsafe way is to install Node in to the same SMP/E environment as LE in order to fulfill the hard prereq. This turned out to be ok for z/OS 2.2, but when I went to install it in to a z/OS 2.3 environment I found I could not fulfill the prereq as z/OS have upped the level of LE and as such changed the FMID to HLE77B0 and in the supplied ADCD code base I use they do not supply the back level LE FMID / environment so I have noway of ever fulfilling the hard prereq in the SMP/E logic, which I am guessing will also cause a number of other people the same problem.

The error you get is as follows:

GIM30209E ** APPLY PROCESSING FAILED FOR SYSMOD HALU600. REQUIRED SYSMODS FAILED
GIM35908I COREQUISITE SYSMOD HLE77A0 WAS MISSING. 
GIM37001S ** APPLY PROCESSING STOPPED BECAUSE FUNCTION SYSMOD HALU600 FAILED. 
PMNode is FMID HALU600

HALU600 is the Node JS FMID which is the one that's failing.

In order to get around this I made a copy of the Node JS  SMPMCS and removed the hard LE prereq and then ran installation jobs to create my own SMP/E environment for Node JS, knowing that the installation would fail as it required parts of LE that I did not supply (as I did not know what parts it needed). The SMP/E failure message told me it needed 2 LE Libraries SCEEBND2 & SCEELIB so I coded up a set of DDDEFS for these 2 libraries and ran the SMP/E setup again and then the Node JS installation and this time it successfully installed.
We should not expect customer to edit the SMP/E installation logic to install Node and we should not have a hard prereq of the LE Libraries supplied in z/OS 2.2. We should supply jobs that allow customers to create the right DDDEFS for whatever level of LE and call by module / library name rather than by FMID.

**Comment by joransiu** [comment]
@stevchen-ibm : Can you comment on this?

**Comment by stevchen-ibm** [comment]
Please refer to this technote: https://www-01.ibm.com/support/docview.wss?uid=ibm10719725
TL;DR:
For V6, please follow the instructions there to update the install sample jobs accordingly.
We will fix this in the future version by separating the LE stubs and removing the hard dependencies on LE in the same SMP/E zone.
